lib/ansible/plugins/inventory/foreman.py

```diff
@@ -166,8 +166,13 @@ def _get_json(self, url, ignore_errors=None):
 
         return self._cache[self.cache_key][url]
 
-    def _get_hosts(self):
-        return self._get_json("%s/api/v2/hosts" % self.foreman_url)
+    def _get_hosts(self, want_params):
+        if want_params:
```

Comment:
Going to nitpick style a bit, but how about:
```python
url = '%s/api/v2/hosts' % self.foreman_url
if want_params:
    url += '?include=all_parameters'
return self._get_json(url)
```

Reply:
Looks good to me.  I actually appreciate the nitpicking.

I had started to change the `_get_json` method to accept something like an `extra_params` argument to pass things like `include=all_parameters`.  This would allow the method to accept additional parameters in the case that the API changes and needs additional items passed in.  Being that we are only dealing with the hosts endpoint, I felt it was overkill to change that method and quickly reverted to the URI format.